styleguide/screens/Patterns/Modifiers/Modifiers.js

```diff
@@ -0,0 +1,425 @@
+      <H level={ 3 } className={ m.mtr }>Rotation</H>
+      <T elm="p" className={ m.mtr }>
+        You can rotate elements simply by providing them with a class that fits the pattern
+        <C>rotate[degrees]</C>. These classes work great when you need to rotate icons, such as them
+        arrow or chevron, allowing you to point them in different directions as required while
+        allowing us to minimise the number of icons available.
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // transform: rotate(90deg);
+          .rotate90
+          // transform: rotate(180deg);
+          .rotate180
+          // transform: rotate(270deg);
+          .rotate270
+        ` }
+        syntax="css"
+      />
+      <H level={ 3 } className={ m.mtr }>Display and position</H>
+      <T elm="p" className={ m.mtr }>
+        These classes allow you to easily change an elements <C>display</C> or
+        <C>position</C> properties.
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // display: inline-block;
+          .dib
+          // display: block;
+          .db
+          // display: inline;
+          .di
+          // position: relative;
+          .pr
+          // position: absolute;
+          .pa
+          // position: static;
+          .ps
+          // position: fixed;
+          .pf
+        ` }
+        syntax="css"
+      />
+      <H level={ 3 } className={ m.mtr }>Vertical Alignment</H>
+      <T elm="p" className={ m.mtr }>
+        Similarly, we have provided classes for easily setting an element’s <C>vertical-align</C>
+        property.
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // vertical-align: top;
+          .valignt
+          // vertical-align: middle;
+          .valignm
+          // vertical-align: bottom;
+          .valignb
+        ` }
+        syntax="css"
+      />
+      <H level={ 3 } className={ m.mtr }>Float</H>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // vertical-align: top;
+          .valignt
+          // vertical-align: middle;
+          .valignm
+          // vertical-align: bottom;
+          .valignb
+        ` }
+        syntax="css"
+      />
+      <H level={ 3 } className={ m.mtr }>Clearfix</H>
+      <T elm="p" className={ m.mtr }>
+        If you need to apply a “clearfix” to an element, you can either use
+        <C>lost-utility: clearfix</C> in your own class, or supply the <C>.cf</C> class.
+      </T>
+      <H level={ 3 } className={ m.mtr }>Width</H>
+      <T elm="p" className={ m.mtr }>
+        You can make elements full width, or set them back to auto:
+      </T>
+      <CodeBlock
+        className={ m.mtr }
+        code={ dedent`
+          // width: 100%
+          .widthFull
+          // width: auto;
+          .widthAuto
+        ` }
+        syntax="css"
+      />
+    </D>
+  </div>
```

Comment:
Could it be useful to link to the file in Github as this is primarily for devs?

Reply:
I totally agree, it would be really useful and especially on this page but I believe we plan to do this for all relevant pages so in the aid of consistency I would be hesitant to add a link here before we settle on a design.